test/realm_test.dart

```diff
@@ -327,20 +340,23 @@ Future<void> main([List<String>? args]) async {
 
       expect(car.make, equals('Tesla'));
 
+      /* Erhh.. generator prohibit changing primary key.. what about realm_core?!
```

Comment:
We should be sure when disabling tests, reviewers might miss this change.

Reply:
core seems to allow it. This needs an investigation for sure.

Reply:
Let us discuss on round-table.. I find it odd that you can change an objects primary key.

Reply:
We never got to this.

Reply:
let's discuss it on the next meeting

Reply:
We decided to keep primary keys as final. Updating primary keys will not be allowed.

Reply:
Fixed

Reply:
👍